include/net/zzhc.rst

```diff
@@ -0,0 +1,68 @@
+.. _lib_zzhc:
+
+China Telecom self-registration (Zi-ZHu-Ce, ZZHC)
+#################################################
+
+The China Telecom self-registration (Zi-ZHu-Ce, zzhc) registers module information to the server when network is available.
+
+Overview
+********
+
+Telecommunication modules sold in China Telecom network requires registration of certain information on the module to the server (zzhc.vnet.cn).
+The ZZHC library implements chapter 6 of the "China Telecom IoT Module Requirements Whitepaper".
+As a background thread launched on boot, it automatically detects the availability of operator network.
+Once internet connectivity is ready and there is a new simcard which is inserted on boot, it proactively uploads IMEI, model number (i.e. nRF9160), modem revision, and the ICCID and IMSI of the simcard connected to nRF9160.
```

Comment:
There are more than two things that are sent, so I would suggest using a bullet point list:
```suggestion
Once the Internet connectivity is ready and a new SIM card is detected on boot, the library proactively uploads the following information to the ZZHC server:
- IMEI
- model number (that is, nRF9160)
- modem revision
- nRF9160 SIM ICCID
- nRF9160 SIM IMSI
```